### PR TITLE
Fix navbar transparency in light mode and some scss linter issues

### PR DESCRIPTION
**Pull request by gmatev**
## Description

Fix navbar transparency in light mode and some scss linter issues

See separate commits for transparency and linter issues

## Does this PR need a docs update or release note?

- [ ] :white_check_mark: Yes, it's included
- [ ] :clock1: Yes, but in a later PR
- [x] :no_entry: No 

## Type of change

- [ ] :sunflower: Feature
- [x] :bug: Bugfix
- [ ] :world_map: Documentation
- [ ] :robot: Test
- [ ] :computer: CI/Deployment
- [x] :hamster: Trivial/Minor

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by ntolia — COMMENTED** [review]
@gmatev Can you please remove the spurious changes?

**Review by ntolia — CHANGES_REQUESTED** [review]
The navbar color has changed in dark mode and needs to be added back. Looks very strange otherwise. Keeping the navbar color constant (like the footer) between dark and light modes would be ideal

**Comment by ntolia** [comment]
> @gmatev Can you please remove the spurious changes?

Ignore this message if the linter actually complains about the previous version of the code.

File: website/src/css/custom.scss [hunk 1]
```diff
@@ -108,10 +109,6 @@ html[data-theme='dark'] {
   --ifm-link-color: #087EA4;
 }
 
-.navbar {
-  background-color: var(--ifm-background-color);
```
Comment (ntolia) [inline website/src/css/custom.scss:112]: If you set `  --ifm-navbar-background-color:` in `[data-theme='dark'] {`, we should be good to go.

Comment (gmatev) [reply website/src/css/custom.scss:112]: I'll fix it

Comment (gmatev) [reply website/src/css/custom.scss:112]: 5a0a21b55348136007c87bb7a0dd7bff65b65b66

**Review by ntolia — APPROVED** [review]

**Pull request merged**